docker-compose.yml

```diff
@@ -17,6 +18,13 @@ services:
       POSTGRES_SERVER: postgres
       POSTGRES_PORT: 5432
       POSTGRES_DB: mealie
+
+      # WORKERS_PER_CORE: 0.5
+      # MAX_WORKERS: 8
+
+      WEB_CONCURRENCY: 2
+    tmpfs:
```

Comment:
Is the tmpfs required? Looks like it's only supported if Docker is running on Linux. Did you look into any alternatives? I'm happy to look into it if you can let me know what it's used for.

Reply:
By default is using `/dev/shm` but the codefactor throwing issue for it. It is use by Gunicorn, you could look at the `gunicorn_conf.py` and search for `worker_tmp_dir`. Maybe could just remove that line then I guess no need to add tmpfs already.

Reply:
Great, I'll look into this after it's merged. Thanks again!

Reply:
Thanks!